bin/capsule.js

```diff
@@ -253,19 +254,49 @@ const siteParamsFromCmdLine = async(ciprojectName) => {
  *
  */
 const mergeConfig = async (site_config, site_config_params, site_config_file) => {
-  let config_params = JSON.parse(site_config_params)
+  let config_params = site_config_params
   let file_params = {}
   let merged_params = {}
   if(site_config_file !== undefined) {
     file_params = await parseJsonConfig(site_config_file)
   }
+  if (typeof config_params === "string") {
+    config_params = JSON.parse(config_params)
+  }
   if (config_params === undefined) {
     config_params = {}
   }
   merged_params = Object.assign({}, file_params, config_params);
   return Object.assign({}, merged_params, site_config);
 }
 
+/**
+ * Read the contents of an S3 file
+ *
+ * @method getS3File
+ *
+ * @param {String} fileName
+ *
+ * @return {String} contents
+ *
+ */
+const getS3File = (bucketName, fileName) => {
```

Comment:
I don't see this being used anywhere?

Reply:
Ahh yeah, this is now dead code. We don't need to read the _redirects file anymore, as the lambda function handles this with param store. I'll remove this!